build.gradle

```diff
@@ -4,8 +4,8 @@ buildscript {
   ext.versions = [
     kotlin: '1.4.10',
     agp: '3.6.1',
-    layoutlib: '4.0.0-alpha01-1a8ff9c', // "should" be similar to versions.agp
-    androidTools: '26.5.0', // agp + 23.0.0 = androidTools
+    layoutlib: '4.0.0-alpha02-b169460', // "should" be similar to versions.agp
+    androidTools: '26.6.0', // agp + 23.0.0 = androidTools
```

Comment:
bumps layoutlib-api to an updated version, due to a change in Bridge.init(...):

https://android.googlesource.com/platform/prebuilts/studio/layoutlib/+/0b01c42f2fa0ebc760f9c92ecd9cca12f8da5af4

https://android.googlesource.com/platform/frameworks/layoutlib/+/ec0913c9da850d6e98850d4a81a3742eabd97d92%5E%21/#F0